content/docs/command-reference/root.md

```diff
@@ -29,22 +30,78 @@ refer to a file in another directory.
 
 ```dvc
 $ dvc root
-
 .
 
 $ cd subdir
 $ dvc root
-
 ..
 ```
 
 ## Example: Referencing files
 
+Simplify file references when working in a subdirectory of a <abbr>DVC
+project</abbr>.
+
 ```dvc
 $ dvc root
-
 ../../../
 
 $ dvc run -d $(dvc root)/data/file.cvs ... \
     python $(dvc root)/scripts/something.py
 ```
+
+## Example: Output references
+
+Simplify output file or directory references.
+
+```dvc
+$ dvc root
+../../../
+
+$ dvc get -o $(dvc root)/root-model.pkl \
+    https://github.com/iterative/example-get-started model.pkl
+```
+
+## Example: Other commands
+
+Simplify other commands when working in a <abbr>DVC project</abbr>.
```

Comment:
OK interesting example. Let's put it all the way at the end please. Also could the title be a little more specific, as well as explaining what we mean by "simplify" (similar to my previous comment).

Reply:
Moved to the end and updated title to be more specific.

Reply:
The title ended up the same as what I changed the 2nd example to haha. So I merged them.